### PR TITLE
Added css to ligin page errors

### PR DESCRIPTION
**Pull request by riteshpushkar**

### CODE REVIEW

File: frontend/.env.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_BACKEND_PATH=http://127.0.0.1:8000
```
Comment (afeefuddin) [inline frontend/.env.txt:1]: Your shouldn't push env files in your commits.

Comment (riteshpushkar) [reply frontend/.env.txt:1]: okk I'll remove my env and push again

Comment (riteshpushkar) [reply frontend/.env.txt]: removed env, still if there's some changes i need to do pls do tell, its my first pr

**Comment by riteshpushkar** [comment]
@afeefuddin I'll try to complete it today itself sorry foe delay

**Pull request closed without merging**